instructors/views.py

```diff
@@ -8,3 +8,5 @@ class InstructorRegister(CreateView):
     model = Instructor
     fields = '__all__'
     template_name = 'instructor_create_profile.html'
+    context_name = 'form'
+    success_url = 'http://localhost:8000/admin/'
```

Comment:
for success_url use reverse('url_name')
do not hardcode the url like you did here.

The perfect solution would be after creating an instructor profile, you must send the user to view the profile you just created.
Instead of success_url, just define the get_absolute_url on the instructor model to send you to the view profile page
and django will send u to the correct page on success if get_absolute_url is defined

also, did you fetch the latest dev before creating this branch like I showed u?
This seems like an old copy of dev

I will reject this pull request .. create a new branch from the latest dev and do the changes I suggested